### PR TITLE
[PBI-1518] Support VMAP repeat ads

### PR DESCRIPTION
**Pull request by dchauOoyala**
- Needs to be merged at the same time as: http://github.services.ooyala.net/Platform/mjolnir/pull/541

### CODE REVIEW

File: js/ad_manager_vast.js [hunk 1]
```diff
@@ -1787,10 +1796,25 @@ OO.Ads.manager(function(_, $) {
...
+        allowMultipleAds: true,
         time: 0,
         position_type: "t",
       };
-      if (adBreak || adBreak.timeOffset) {
+      if (!adBreak) {
+        return null;
+      }
+      if (adBreak.repeatAfter) {
+        adObject.repeatAfter = _convertTimeStampToSeconds(adBreak.repeatAfter) / 1000;
```
Comment (dchauOoyala) [inline js/ad_manager_vast.js]: Change name of helper function to Milliseconds

File: js/ad_manager_vast.js [hunk 2]
```diff
@@ -1820,26 +1828,25 @@ OO.Ads.manager(function(_, $) {
...
      * @param {string} timeString The timestamp string (format: hh:mm:ss / hh:mm:ss.mmm)
      * @returns {number} The number of milliseconds the timestamp represents.
      */
-    var _convertTimeStampToSeconds = _.bind(function(timeString) {
+    var _convertTimeStampToMilliseconds = _.bind(function(timeString) {
       var hms = timeString.split(":");
       // + unary operator converts string to number
-      // Use parseInt to truncate decimal
-      var seconds = (+hms[0]) * 60 * 60 + (+hms[1]) * 60 + (parseInt(hms[2])) * 1000;
+      var seconds = (+hms[0]) * 60 * 60 + (+hms[1]) * 60 + (+hms[2]) * 1000;
```
Comment (pilievOoyala) [inline js/ad_manager_vast.js]: you should double check that the resulting array is the correct length.

Comment (pilievOoyala) [inline js/ad_manager_vast.js]: Do it in next checkin.

**Comment by pilievOoyala** [comment]
LGTM.

**Pull request merged**